Week1_final.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Similar to previous comment, isolating the equations would make it easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>